src/chat/UnreadNotice.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
Sometimes in future, we might want to make all Notice components into one configurable component? Just a food for thought. It is fine as-is.